### ISSUE TITLE
New warning color doesn't meet contrast

### ISSUE THREAD

**Issue by dialpad-drew** [dialpad/dialtone#336]
New colors applied to our warning validation doesn't meet [contrast requirements](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast). 

![image](https://user-images.githubusercontent.com/77751297/119876687-610d9380-bee5-11eb-99cc-d1da7eca401e.png)
